apps/mobile/src/navigation/main-router.tsx

```diff
@@ -0,0 +1,75 @@
+import { Ionicons } from "@expo/vector-icons";
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import { NavigationContainer } from "@react-navigation/native";
+import { StatusBar } from "expo-status-bar";
+import { Platform } from "react-native";
+import { ToastProvider } from "~/components/toast-provider";
+import { BleStackNavigator } from "~/navigation/ble-stack-navigator";
+import { HomeScreen } from "~/screens/home-screen";
+import { SerialPortConnectionScreen } from "~/screens/serial-port-connection-screen";
+
+import { BluetoothStackNavigator } from "./bluetooth-stack-navigator";
+
+const Tab = createBottomTabNavigator();
+
+export function MainRouter() {
```

Comment:
Calling a separate renderMainRouter() function in JSX adds unnecessary indirection. It’s clearer to inline the <Tab.Navigator> directly in the return.